apps/web/src/hooks/use-create-stream.test.ts

```
@@ -0,0 +1,117 @@
+
+const streamInput = {
+  recipient: 'GRECIPIENT',
+  token: 'CTOKEN',
+  amount: 100n,
+  startTime: 1_700_000_000,
+  endTime: 1_700_003_600,
+};
+
+function createWrapper(queryClient: QueryClient) {
+  return ({ children }: { children: ReactNode }) =>
+    createElement(QueryClientProvider, { client: queryClient }, children);
```

ESLint

[error] Component definition is missing display name

(react/display-name)